### PR TITLE
feat: APP-2619 - Implement Card and CardSummary components

### PR DESCRIPTION
**Pull request by RuggeroCino**
## Description

- Implement `Card` and `CardSummary` components

Task: [APP-2619](https://aragonassociation.atlassian.net/browse/APP-2619)

## Type of change

-   [x] New feature (non-breaking change which adds functionality)

## Checklist:

-   [x] I have selected the correct base branch.
-   [x] I have performed a self-review of my own code.
-   [x] I have commented my code, particularly in hard-to-understand areas.
-   [x] I have made corresponding changes to the documentation.
-   [x] My changes generate no new warnings.
-   [x] Any dependent changes have been merged and published in downstream modules.
-   [x] I ran all tests with success and extended them if necessary.
-   [x] I have updated the `CHANGELOG.md` file in the root folder of the package after the [UPCOMING] title and before
        the latest version.
-   [x] I have tested my code on the test network.

[APP-2619]: https://aragonassociation.atlassian.net/browse/APP-2619?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/cards/cardSummary/cardSummary.api.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import type { IconType } from '../../icon';
+import type { ICardProps } from '../card/card';
+
+export interface ICardSummaryAction {
```
Comment (Fabricevladimir) [inline src/components/cards/cardSummary/cardSummary.api.ts:4]: Why narrow this to merely the label and onClick? Do we anticipate that we won't need to change/remove the button icon for example?

File: src/components/cards/cardSummary/cardSummary.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            'grid-cols-[auto_max-content] md:grid-flow-col md:grid-cols-[auto_1fr_1fr_max-content] md:gap-6':
+                !isStacked,
+        },
+    );
+
+    return (
+        <Card
+            className={classNames(
+                { 'w-[320px] md:w-[480px]': isStacked },
+                { 'w-[320px] md:w-[640px]': !isStacked },
```
Comment (Fabricevladimir) [inline src/components/cards/cardSummary/cardSummary.tsx:24]: Should the width be fixed or does it take the size of the parent container?

File: src/components/cards/cardSummary/cardSummary.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                </Button>
+                <div
+                    className={classNames(
+                        'col-span-2 flex gap-x-2 gap-y-1',
+                        { 'flex-col': isStacked },
+                        { 'flex-col md:col-start-2 md:flex-row md:items-baseline': !isStacked },
+                    )}
+                >
+                    <p className="text-2xl font-semibold leading-tight text-neutral-800 md:text-3xl">{value}</p>
+                    <p className="text-base font-normal leading-tight text-neutral-400">{description}</p>
```
Comment (Fabricevladimir) [inline src/components/cards/cardSummary/cardSummary.tsx:54]: No changes needed, but a little TW tip. Apparently you can merge the font size and line height like so: `text-base/tight`

**Review by Fabricevladimir — COMMENTED** [review]
Looking good! Thanks

File: src/components/cards/cardSummary/cardSummary.api.ts (hunk 1 above)
Comment (RuggeroCino) [reply src/components/cards/cardSummary/cardSummary.api.ts:4]: Exactly, currently only the `label` will be customisable and the icon will always be displayed. We can change it later if requirements will change :)

File: src/components/cards/cardSummary/cardSummary.tsx (hunk 2 above)
Comment (RuggeroCino) [reply src/components/cards/cardSummary/cardSummary.tsx:24]: All these containers have a fixed size set on Figma

File: src/components/cards/cardSummary/cardSummary.tsx (hunk 3 above)
Comment (RuggeroCino) [reply src/components/cards/cardSummary/cardSummary.tsx:54]: Good to know :D

File: src/components/cards/cardSummary/cardSummary.tsx (hunk 2 above)
Comment (Fabricevladimir) [reply src/components/cards/cardSummary/cardSummary.tsx:24]: Yeah, I'm asking myself if that's the actual card size or if this is just Selim needing to limit it to a size on Figma 🤔

**Review by Fabricevladimir — APPROVED** [review]

**Pull request merged**